### PR TITLE
Fix cursor issue with search box

### PR DESCRIPTION
**Pull request by sgrg1**
Ready for PR

### CODE REVIEW

**Comment by martypowell** [comment]
> Ready for PR

When submitting a PR, please submit what issue you are addressing and a brief description of the problem.

Example:

Addresses #606 

Focuses cursor on search input when when the search tab is selected.

File: js/site-search.js [hunk 1]
```diff
@@ -134,13 +134,14 @@ bcpl.siteSearch = (($, window, constants, querystringer) => {
 		const buttonCaption = $searchBtn.find('i span').text().trim();
 
 		$(siteSearchInputSelector).attr('placeholder', `${buttonCaption}`);
+
```
[the remark this answers is not in the document]
Comment (tmccoy529) [reply js/site-search.js]: Something happened and Sanjay's branch is no longer connected to git. Anyway I removed this space and pushed back up.

**Comment by sgrg1** [comment]
Declare  variable name at the top of the function per marty. Will move to dev once issue  #655 is approve

File: js/site-search.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (martypowell) [reply js/site-search.js]: @tmccoy529 looks like there is still one formatting issue.

image

[the remark this answers is not in the document]
Comment (martypowell) [reply js/site-search.js]: only reason, im harping on this, is because there were no changes to this file.

**Comment by martypowell** [comment]
@tmccoy529 this is approved, and can be moved to dev for testing.

**Pull request merged**